easybuild/easyconfigs/s/seq2HLA/seq2HLA-20180802-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'Tarball'
+
+name = 'seq2HLA'
+version = '20180802'
```

Comment:
@smoors I think it makes sense to go with `2.3` here, since that's what this (fairly old) commit clearly points to.

Reply:
@boegel fine by me, I'm just a little worried that the code for this version might change because it is not a real release.

Reply:
@smoors Note that in #6965 I'm also installing by commit ID, so it's "safe".